### PR TITLE
ADAPTTSSS-706 | @rebeccahongsf | refresh public api keys

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- refresh public api keys

# Review By (Date)
- asap

# Review Tasks

## Setup tasks and/or behavior to test

1. Navigate to https://saaassociates-test.sites.stanford.edu/ 
2. Verify that the react widget renders and works
3. Review code

# Associated Issues and/or People
- ADAPTSSS-706

### CODE REVIEW

File: src/associates/lib/main.css [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-
+#su-associates .name{margin-left:5%;padding:.3em 0;display:inline-block;width:40%}#su-associates .degrees{margin-left:5%;display:inline}
+body{color:#726543;font-size:13px}.newMembersToggle{margin-top:2em;margin-left:3%}.nav-tabs .nav-item.active{background-color:#726543;color:#fff}.nav-tabs .nav-item{background-color:#dcdac1;color:#726543;border:1px solid #c1bda2;border-radius:0}.tab-pane{background-color:#f3f1ed;border-color:#dddbc3}#su-associates .nav{margin-top:3em}#su-associates h5{width:95%;color:#7d9a18;border-bottom:1px dashed #e0ddd6;padding:10px;margin:0 auto 20px auto}#su-associates .topLink{color:#8c1515;float:right;margin-right:2%;margin-top:-1.7em}
```
Comment (yvonnetangsu) [inline src/associates/lib/main.css:5]: Looks like it just removes the empty lines here @rebeccahongsf yeah? 😄

**Review by yvonnetangsu — APPROVED** [review]
Widget works and LGTM! 😄 🎉

**Review by yvonnetangsu — APPROVED** [review]
Widget works and LGTM! 😄 🎉

File: src/associates/lib/main.css (hunk 1 above)
Comment (rebeccahongsf) [reply src/associates/lib/main.css:5]: @yvonnetangsu oh yeah 😆

**Pull request merged**